cmd/bake/bake.go

```diff
@@ -9,8 +9,10 @@ import (
 	"fmt"
 	"io"
 	"net/http"
+	"os"
 
 	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
```

Comment:
I typically like to seperate blocks of imports by their starting import path (i.e., group all the `github.com` ones together, group all the `gopkg` ones together, etc.). Something like this should suffice:

```suggestion
	"github.com/spf13/cobra"

	"gopkg.in/yaml.v3"
```